monai/bundle/utils.py

```diff
@@ -113,7 +113,7 @@
         "experiment_name": "monai_experiment",
         "run_name": None,
         # may fill it at runtime
-        "execute_config": None,
+        "save_execute_config": True,
```

Comment:
As the `DEFAULT_MLFLOW_SETTINGS` is more and more complicated, I would suggest to write a clear doc-string for the description of every parameter here and add to the API-DOC: https://docs.monai.io/en/latest/bundle.html.

Reply:
Ok will update the doc.

Reply:
Done